CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Kubernetes API Version: 1.16.14
 
 Kubernetes API Version: 1.16.14
 
+**Breaking Change:**
+- `kubernetes.config.Configuration()` no longer returns the default configuration. Use `kubernetes.config.Configuration.get_default_copy()` now to get the default configuration. [OpenAPITools/openapi-generator#4485](https://github.com/OpenAPITools/openapi-generator/pull/4485), [OpenAPITools/openapi-generator#5315](https://github.com/OpenAPITools/openapi-generator/pull/5315)
```

Comment:
it's a bit confusing ...

kubernetes.config.Configuration() still returns the default "initial" configuration,
kubernetes.config.Configuration.get_default_copy() will return the default configuration if there is a default set via Configuration.set_default(c), otherwise, it will also return the default "initial" configuration

Reply:
Updated. I reworded it a little bit, since the change in `kubernetes.config.Configuration()` was the regression.